aten/src/ATen/native/cuda/LegacyDefinitions.cpp

```diff
@@ -69,4 +69,33 @@ Tensor gather_cuda(const Tensor & self, int64_t dim, const Tensor & index, bool
   return gather_out_cuda(result, self, dim, index, sparse_grad);
 }
 
+Tensor & fmod_cuda_out(Tensor & result, const Tensor & self, Scalar other) {
+  return legacy::cuda::_th_fmod_out(result, self, other)
+}
+
+Tensor fmod_cuda(const Tensor & self, Scalar other) {
+  return legacy::cuda::_th_fmod(self, other);
+}
+
+Tensor & fmod_cuda_out(Tensor & result, const Tensor & self, const Tensor & other) {
+  std::tie(b_self, b_other) = expand_outplace(self, other, "fmod_out");
```

Comment:
missing decls for these I think

Reply:
yep, I didn't copy paste enough :).

Reply:
and I usually don't actually build/test cuda because it takes too long...